packages/wonder-blocks-form/components/radio.js

```diff
@@ -27,6 +27,13 @@ type ChoiceComponentProps = {|
      */
     onChange: (newCheckedState: boolean) => void,
 
+    /**
+     * Optional label if it is not obvious from the context what the radio
+     * does. If the label and id props are defined, this props does not need to
+     * be provided as the label would be matched to this input.
+     */
+    ariaLabel?: string,
```

Comment:
Let's make this `aria-label`.

Reply:
I just noticed that IconButton uses `aria-label`. Some components in `webapp`, though, use `ariaLabel`. Maybe we should add something to the style guide 🤔

Reply:
I think the disadvantage of `aria-label` is that we can't unpack it in the regular props unpacking part.

Reply:
Is bumping up the version a matter of manually editing `version` 2.0.1 => 3 in `package.json`?

Reply:
We'll bump when we do a release.  Can you edit the title of PR to include `BREAKING CHANGE`.  It would be cool if we could use https://github.com/semantic-release/semantic-release, but currently it doesn't work with `lerna` so we'll have to track this manually for now.  cc @jeresig

Reply:
Okay... so there's actually a way to destructure objects with `aria-label` in it... by renaming it in the destructuring assignment, e.g.
```
const {
   'aria-label': ariaLabel,
   ...otherProps,
} = props;
```